client/app/bundles/HelloWorld/containers/HelloWorld.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { PropTypes } from 'react';
+import HelloWorldWidget from '../components/HelloWorldWidget';
+import { connect } from 'react-redux';
+import Immutable from 'immutable';
```

Comment:
@nostophilia We could. @robwise and @alexfedoseev Please take a look at the code that's generated here.